src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -1166,6 +1180,7 @@ query_path(const char *szInput, int *is_target_path, struct dcache_rec **parent,
 			}
 
 			rc = daos_init();
+			D_DEBUG(DB_ANY, "called daos_init().\n");
```

Comment:
this debug statement looks unnecessary, please remove.

Reply:
ok. I will remove it in next commit. Thank you!

Reply:
The debug message is used by a ftest. I added a comment. Thank you!

Reply:
This debug message for users is useless and will just cause clutter in the debug logs.
if you have a test relying on this message to see if daos_init() is called, you need to update the test.
How about you check what currently daos_init() prints when it's called in INFO and use those in the test?
If there is nothing clear there you can add something as an INFO message in daos_init() itself, not here, towards printing something like:
libdaos-"version" initializing
this does provide something meaningful for users (the version).

Reply:
ok. Thank you! I will remove this line and check for existing debug output, e.g., "crt_init_opt() libcart".

Reply:
Fixed as you suggested. Thank you very much!